### PR TITLE
update capacities India

### PR DESCRIPTION
**Pull request by KabelWlan**
I updated the capacities of India to March 2021. I used the sources listed in data_sources.md.

### CODE REVIEW

**Comment by madsnedergaard** [comment]
@pierresegonne, would you mind reviewing this? I see that there are some changes to IN-UP since [your PR that updated capacities](https://github.com/tmrowco/electricitymap-contrib/pull/3214), so would be nice to know why and if there's other factors of interest (for example I find it weird that Delhi is reported as 0 coal, but maybe I missed something?)

**Comment by pierresegonne** [comment]
Hi @KabelWlan Thank you for your contribution!

It nevertheless seems like you are not using the latest released capacities for the update.

Please find them here:
[INDIA](https://npp.gov.in/public-reports/cea/monthly/installcap/2021/JUN/capacity1-2021-06.pdf)
[Northern](https://npp.gov.in/public-reports/cea/monthly/installcap/2021/JUN/capacity2-Northern-2021-06.pdf)
[Eastern](https://npp.gov.in/public-reports/cea/monthly/installcap/2021/JUN/capacity2-Eastern-2021-06.pdf)
[Western](https://npp.gov.in/public-reports/cea/monthly/installcap/2021/JUN/capacity2-Western-2021-06.pdf)
[Southern](https://npp.gov.in/public-reports/cea/monthly/installcap/2021/JUN/capacity2-Southern-2021-06.pdf)
[North-Eastern](https://npp.gov.in/public-reports/cea/monthly/installcap/2021/JUN/capacity2-North%20Eastern-2021-06.pdf)

**Comment by KabelWlan** [comment]
I used exactly those sheets for everything except of solar, wind and biomass since they're combined in the category "RES". I used the following pages:
[solar](https://www.mnre.gov.in/solar/current-status/) 
[wind](https://www.mnre.gov.in/wind/current-status) 
[biomass](https://www.mnre.gov.in/bio-energy/current-status) 
[waste](https://www.mnre.gov.in/waste-to-energy/current-status) which I included to biomass

However after going through your PR I found out that I should better use [this page](https://npp.gov.in/dashBoard/cp-map-dashboard) to split up the renewables and get the latest data...

Thanks for mentioning, learned something new again, also sorry for the circumstances caused... 
I'll change this now and also update the data source with the direct link to the dashboard and I'll delete the "outdated" sources.

File: config/zones.json [hunk 1]
```diff
@@ -2661,13 +2661,13 @@
       ]
     ],
     "capacity": {
-      "biomass": 0,
-      "coal": 4404,
-      "gas": 2115,
-      "hydro": 723,
-      "nuclear": 103,
+      "biomass": 52,
+      "coal": 0,
```
Comment (pierresegonne) [inline config/zones.json:2665]: Interesting that Delhi has no coal power station anymore ! [Supporting document](https://cea.nic.in/wp-content/uploads/pdm/2020/09/list_power_stations_2020.pdf)

**Review by pierresegonne — APPROVED** [review]
Great job, thanks!

**Pull request merged**